### PR TITLE
Content Pack Installer docs update

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-30333)

## Description
After a bug was opened due to misinterpreted script usage, I update the docs to be compatible with our regular template along with extending the troubleshoot section.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/ContentPackInstaller/README.md [hunk 1]
```diff
@@ -1,7 +1,60 @@
-### Command Example
+Install Content packs from marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/ContentPackInstaller/README.md:1]:
```suggestion
Install content packs from Marketplace.
```

File: Packs/CommonScripts/Scripts/ContentPackInstaller/README.md [hunk 2]
```diff
@@ -1,7 +1,60 @@
...
+#### With escape chars
+`!ContentPackInstaller packs_data="[{\"packid\": \"GoogleCloudCompute\",\"packversion\": \"latest\"}]" pack_id_key=packid pack_version_key=packversion`
+
 ### Troubleshooting
+#### Mulit-tenant environments 
 Multi-tenant environments should be configured with the Cortex Rest API instance when using this 
 automation. Make sure the *Use tenant* parameter (in the Cortex Rest API integration) is checked 
 to ensure that API calls are made to the current tenant instead of the master tenant.
+
+#### General Failures
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/ContentPackInstaller/README.md:55]: Do you mean Troubleshooting?

File: Packs/CommonScripts/Scripts/ContentPackInstaller/README.md [hunk 3]
```diff
@@ -1,7 +1,60 @@
...
+1. Make sure the Core REST API integration is configured correctly, and the pressing the test button returns success.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/ContentPackInstaller/README.md:56]:
```suggestion
- Make sure the Core REST API integration is configured correctly, and clicking the test button returns success.
```

File: Packs/CommonScripts/Scripts/ContentPackInstaller/README.md [hunk 4]
```diff
@@ -1,7 +1,60 @@
...
+2. Make sure you have a connection to the marketplace, you can check that by going to the marketplace page in the XSOAR UI and refreshing. 
+3. Make sure the *packs_data* parameter value is in the correct format:
+   1. A list `[]` containing dictionaries `{}`. For each pack you want to install, add a dictionary with pack_id_key as the pack id key and pack_version_key as the pack version key as demonstrated in the command examples above.
+   2. The escape characters are in the right order and none are missing.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/ContentPackInstaller/README.md:60]:
```suggestion
- Make sure you have a connection to Marketplace. You can check that by going to the Marketplace page in the Cortex XSOAR UI and refreshing. 
- Make sure the *packs_data* parameter value is in the correct format:
   - A list `[]` containing dictionaries `{}`. For each pack you want to install, add a dictionary with pack_id_key as the pack id key and pack_version_key as the pack version key as demonstrated in the command examples above.
   - The escape characters are in the right order and none are missing.
```

**Comment by ShirleyDenkberg** [comment]
@ShahafBenYakir Doc review completed.

File: Packs/CommonScripts/Scripts/ContentPackInstaller/README.md [hunk 5]
```diff
@@ -1,7 +1,60 @@
...
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| ContentPackInstaller.packname | The name of the pack. | Unknown |
+| ContentPackInstaller.packversion | The version of the pack. | Unknown |
+| ContentPackInstaller.installationstatus | The installation status of the pack. | Unknown |
+
+### Command Examples
+#### Without escape chars
 !ContentPackInstaller packs_data=`[{"id":"GoogleCloudCompute","itemVersion":"latest"},{"id":"GoogleKubernetesEngine","itemVersion":"latest"},{"id":"GoogleSafeBrowsing","itemVersion":"latest"}]` pack_id_key=id pack_version_key=itemVersion
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/Scripts/ContentPackInstaller/README.md:44]:
```suggestion
``!ContentPackInstaller packs_data=`[{"id":"GoogleCloudCompute","itemVersion":"latest"},{"id":"GoogleKubernetesEngine","itemVersion":"latest"},{"id":"GoogleSafeBrowsing","itemVersion":"latest"}]` pack_id_key=id pack_version_key=itemVersion``
```

File: Packs/CommonScripts/Scripts/ContentPackInstaller/README.md [hunk 6]
```diff
@@ -1,7 +1,60 @@
...
+| Cortex XSOAR Version | 6.0.0 |
+
+## Used In
+
+---
+This script is used in the following playbooks and scripts.
+
+* Configuration Setup
+* Content Update Manager
+
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/Scripts/ContentPackInstaller/README.md:20]: Was this generated automatically?

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by DinaMeylakh** [comment]
@ShahafBenYakir Force merge needed: failing on unittests coverage of ContentPackInstaller where unittests file doesn't exit. As this pr only changes documentation, it is irrelevant.

**Pull request merged**